pkg/email/parser.go

```diff
@@ -97,6 +98,14 @@ func Parse(r io.Reader, ownEmails []string) (*Email, error) {
 		}
 	}
 	ccList = MergeEmailLists(ccList)
+
+	listIDs, _ := msg.Header.AddressList("List-ID")
```

Comment:
Looking at one of emails in my inbox I see: `List-ID: <syzkaller-bugs.googlegroups.com>`
It's not an email address. Will it be parsed correctly?

Reply:
Good catch! It won't fail parsing as I ignored parsing errors for ListID in the code, but the recognition just won't work.

There's a [List-Post](https://www.rfc-editor.org/rfc/rfc2369#section-3.4) that looks more reasonable, but now I'm thinking that just the normal `Sender` field might suffice. At least it's definitely the case for googlegroups.com.

Adjusted the code.